jacobian_follower/src/jacobian_follower/jacobian_follower.cpp

```diff
@@ -274,11 +274,14 @@ std::optional<moveit_msgs::RobotState> JacobianFollower::computeCollisionFreePoi
     ROS_DEBUG_STREAM_NAMED(LOGGER_NAME + ".ik",
                            "goal:  [" << name << "] positions " << p.x << "," << p.y << "," << p.z);
   }
+  opts.goals.emplace_back(std::make_unique<bio_ik::MinimalDisplacementGoal>());
 
   robot_state::RobotState robot_state_ik(model_);
+  robot_state::RobotState seed_robot_state_ik(model_);
   ROS_DEBUG_STREAM_NAMED(LOGGER_NAME + ".ik", "" << default_robot_state.joint_state.name.size() << " "
                                                  << default_robot_state.joint_state.position.size());
-  auto const success = moveit::core::robotStateMsgToRobotState(default_robot_state, robot_state_ik);
+  auto success = moveit::core::robotStateMsgToRobotState(default_robot_state, robot_state_ik);
```

Comment:
would be cleaner to make `auto const success = moveit::core::robotStateMsgToRobotState(default_robot_state, robot_state_ik) && moveit::core::robotStateMsgToRobotState(default_robot_state, seed_robot_state_ik);`